### PR TITLE
Create PUT operation to update a user

### PR DESCRIPTION
**Pull request by QAMemo**

### CODE REVIEW

File: src/test/java/com/at/stepdefinitions/Put.java [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+        jsonBodyRequest.put("email", lorem.getFirstName().toLowerCase()+"@gmail.com");
+        jsonBodyRequest.put("password", lorem.getFirstName().toLowerCase()+"12345678");
+        jsonBodyRequest.put("status", 0);
+        base.requestBody = jsonBodyRequest.toString();
+        base.apiResource = "/api/v1/users";
+    }
+
+    @Given("I get the id of a new user")
+    public void i_get_a_id_user_randomly() {
+    	//base.apiResource = "/api/v1/users";
```
Comment (VillaAT) [inline src/test/java/com/at/stepdefinitions/Put.java]: Try to clean your code before the commit.

File: src/test/java/com/at/stepdefinitions/Put.java [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+        Lorem lorem = LoremIpsum.getInstance();
+        JSONObject jsonBodyRequest = new JSONObject();
+        jsonBodyRequest.put("type", 1);
+        jsonBodyRequest.put("firstName", lorem.getFirstName().toLowerCase());
+        jsonBodyRequest.put("lastName", lorem.getLastName().toLowerCase());
+        jsonBodyRequest.put("email", lorem.getFirstName().toLowerCase()+"@gmail.com");
+        jsonBodyRequest.put("password", lorem.getFirstName().toLowerCase()+"12345678");
+        jsonBodyRequest.put("status", 0);
+        base.requestBody = jsonBodyRequest.toString();
+        base.apiResource = "/api/v1/users";
```
Comment (VillaAT) [inline src/test/java/com/at/stepdefinitions/Put.java]: use Api path reference.

File: src/test/java/com/at/stepdefinitions/Put.java [hunk 3]
```diff
@@ -40,13 +40,18 @@ public void create_body_for_a_new_user(){
         jsonBodyRequest.put("email", lorem.getFirstName().toLowerCase()+"@gmail.com");
         jsonBodyRequest.put("password", lorem.getFirstName().toLowerCase()+"12345678");
         jsonBodyRequest.put("status", 0);
+        
         base.requestBody = jsonBodyRequest.toString();
-        base.apiResource = "/api/v1/users";
+        for(ApiPath a: ApiPath.values()) {
```
Comment (VillaAT) [inline src/test/java/com/at/stepdefinitions/Put.java]: Did you try using ApiPath.updating_users.getApiPath(); instead of using the for statement?

**Review by ramiroAT — APPROVED** [review]

**Review by VillaAT — APPROVED** [review]
Good Luck!

**Pull request merged**